packages/ui/components/form/switch/Switch.tsx

```diff
@@ -29,40 +30,76 @@ const Switch = (
 
   return (
     <Wrapper tooltip={props.tooltip}>
-      <div
-        className={cx(
-          "flex h-auto w-auto flex-row items-center",
-          fitToHeight && "h-fit",
-          classNames?.container
-        )}>
-        <PrimitiveSwitch.Root
+      {props.isRoutingForm ? (
```

Comment:
I don't know why we need `isRoutingForm` here. It is a generic Switch component which shouldn't know what a Routing FOrm is